Assets/Scripts/Nathan/Animator/AnimatorMovement.cs

```diff
@@ -0,0 +1,126 @@
+using UnityEngine;
+
+public class AnimatorMovement : MonoBehaviour
+{
+    public GameObject playerGhost;
+
+    private Animator animator;
+    private Rigidbody2D playerRigidBody;
+    private Vector2 grabPoint;
+    private Vector2 grabPointOffset;
+    private bool grabbedChanged;
```

Comment:
Could this value be handled by the animator as well?

Reply:
No, that value is for specific script behaviors that happen for one iteration of update. grabbedChanged acts as a one way flag.